webapp/packages/core-ui/src/Tabs/Tab/Tab.tsx

```diff
@@ -8,71 +8,69 @@
 import { useTab } from './useTab';
 
-export const Tab = observer<TabProps>(function Tab({ tabId, title, menuContext, disabled, className, children, style, onOpen, onClose, onClick }) {
+export const Tab = observer<TabProps>(function Tab({ tabId, title, menuContext, disabled, className, children, onOpen, onClose, onClick }) {
   const translate = useTranslate();
   const ref = useRef<HTMLButtonElement>(null);
   const tabContext = useMemo(() => ({ tabId }), [tabId]);
   const tab = useTab(tabId, onOpen, onClose, onClick);
   const info = tab.getInfo();
+  const styles = useS(baseTabStyles, baseTabActionStyles);
 
   const canClose = getComputed(() => !!onClose || (tab.closable && tab.state.closable));
 
-  return styled(useStyles(BASE_TAB_STYLES, BASE_TAB_ACTION_STYLES, style))(
+  return (
     <TabContext.Provider value={tabContext}>
-      <tab-outer>
-        <tab-inner {...use({ selected: tab.selected })}>
-          <tab-actions>
+      <div className={s(styles, { tabOuter: true })}>
+        <div className={s(styles, { tabInner: true, tabInnerSelected: tab.selected })}>
+          <div className={s(styles, { tabActions: true })}>
             {canClose && (
-              <tab-action title={translate('ui_close')} onClick={tab.handleClose}>
-                <Icon name="cross-bold" viewBox="0 0 7 8" />
-              </tab-action>
+              <div className={s(styles, { tabAction: true })} title={translate('ui_close')} onClick={tab.handleClose}>
+                <Icon className={s(styles, { tabIcon: true })} name="cross-bold" viewBox="0 0 7 8" />
+              </div>
             )}
-            <TabMenu tabId={tabId} state={tab.state} menuContext={menuContext} style={style} />
-          </tab-actions>
+            <TabMenuNew tabId={tabId} state={tab.state} menuContext={menuContext} />
+          </div>
           <BaseTab
             ref={ref}
             {...tab.state.state}
             type="button"
             title={translate(title ?? info?.title)}
             id={tabId}
-            className={className}
+            className={s(styles, { baseTab: true }, className)}
             disabled={disabled}
             onClick={tab.handleOpen}
           >
-            <tab-container>{children}</tab-container>
+            <div className={s(styles, { tabContainer: true })}>{children}</div>
           </BaseTab>
-        </tab-inner>
-      </tab-outer>
-    </TabContext.Provider>,
+        </div>
+      </div>
+    </TabContext.Provider>
   );
 });
 
 interface TabMenuProps {
   tabId: string;
   state: ITabsContext<any>;
   menuContext?: IDataContext;
-  style?: ComponentStyle;
 }
 
-const TabMenu = observer<TabMenuProps>(function TabMenu({ tabId, state, menuContext, style }) {
-  const styles = useStyles(BASE_TAB_STYLES, BASE_TAB_ACTION_STYLES, style);
+const TabMenuNew = observer<TabMenuProps>(function TabMenuNew({ tabId, state, menuContext }) {
+  const styles = useS(baseTabStyles, baseTabActionStyles);
```

Comment:
probably you want to keep old name for this component `TabMenu`

Reply:
done